frontend/src/CopyValue.js

```diff
@@ -17,15 +17,21 @@ const Hover = styled.span`
 `;
 
 class CopyValue extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {isStricken: false};
+  }
+
   onClick = () => {
     copy(this.props.text);
+    this.setState({isStricken: !this.state.isStricken});
```

Comment:
If new state depends on previous state, use update function instead, it guarantees race-condition free code! (`setState` is asynchronous)

```js
this.setState(state => ({ 
    isStricken: !state.isStricken 
}));
```